components/footer.js

```diff
@@ -8,7 +8,8 @@ import { injectIntl, intlShape } from 'react-intl'
 const StyledFooter = styled.footer`
   background-color: ${props => props.theme.colors.gray1};
   color: ${props => props.theme.colors.blue9};
-  font-size: 0.8rem;
+  font-size: 0.857rem;
```

Comment:
What is the origin of this odd font-size?

Reply:
When I first wrote the footer, the font size looked a bit big for smaller screens and so ended up using the `rem` notation to keep it relative. When @elioqoshi was reviewing it he found the font-size as `11.2px` where he expected `12px`. But he also agreed that using rem makes sense. Hence this value.